test/functional/aws_standalone_vpc.go

```diff
@@ -160,18 +170,49 @@ func TestStandaloneVPCExists(t *testing.T, testingCtx *common.TestingContext) {
 		t.Skip("_network key does not exist in aws strategy configmap, skipping standalone vpc network test")
 	}
 
-	// get the vpc cidr block
-	expectedCidr, err := getCidrBlock(strat)
-	if err != nil {
-		t.Fatal("could not get cidr block", err)
-	}
-
 	// get the cluster id used for tagging aws resources
 	clusterTag, err := getClusterID(ctx, testingCtx.Client)
 	if err != nil {
 		t.Fatal("could not get cluster id", err)
 	}
 
+	// get the vpc cidr block
+	expectedCidr, err := getCidrBlockFromStrategyMap(strat)
+	if err != nil {
+		t.Fatal("could not get cidr block from strategy map", err)
+	}
+
+	vpcCidrBlock := expectedCidr
```

Comment:
this statement feels a bit funny to me. I guess the goal with it is to have something inside `vpcCidrBlock` lower down in the code and that would be reassigned `if expectedCidr == ""`

I think that it would flow a bit better if this was removed and then the `expectedCidr = vpcCidrBlock` was added just before the close of the `if expectedCidr == ""`

You could keep `expectedCidr` as the param in the lower down functions.

Reply:
Agreed. Great suggestion. Makes it a lot cleaner.